plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -226,6 +249,12 @@ class account_failures {
 
    std::map<account_name, account_failure> failed_accounts;
    uint32_t max_failures_per_account = 3;
+   mutable fc::microseconds block_idle_time;
```

Comment:
Quite unusual to see this. How bad is it to unwind the const train to not make this mutable? A cursory look, seems like `report() const` is called from both `abort_block()` & `produce_block()` (neither of which are const already)

Reply:
Thanks. Removed mutable/const which is much cleaner.